dvc/command/pkg.py

```diff
@@ -48,42 +71,46 @@ def add_parser(subparsers, parent_parser):
     PKG_INSTALL_HELP = "Install package."
     pkg_install_parser = pkg_subparsers.add_parser(
         "install",
-        parents=[parent_config_parser, parent_parser],
+        parents=[parent_parser],
         description=append_doc_link(PKG_INSTALL_HELP, "pkg-install"),
         help=PKG_INSTALL_HELP,
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
+    pkg_install_parser.add_argument("url", help="Package URL.")
+    pkg_install_parser.add_argument("--version", help="Package version.")
     pkg_install_parser.add_argument(
-        "address",
-        nargs="?",
-        default="",
-        help="Package address: git://<url> or https://github.com/...",
+        "--name",
+        help=(
+            "Package alias. If not specified, the name will be determined "
```

Comment:
just: "Package URL or alias."

Reply:
You don't specify URL as a `--name`. Current description is telling that you can specify the alias, but if not than the name is going to be determined from URL that is specified separately. This description makes sense to me.

Reply:
Ah, got it. You were talking about API CLI mismatch. Fixing.